src/build/env.py

```diff
@@ -185,18 +186,12 @@ def _create_env_pythonhome(self):  # type: () -> None
     def _create_env_venv(self):  # type: () -> None
         if sys.version_info[0] == 2:
             raise RuntimeError('venv not available on Python 2')
-        else:  # make mypy happy
-            if os.name == 'nt' and sys.version_info[0] == 3 and sys.implementation.name == 'pypy':
-                # ensurepip is borked, bring your own pip
-                venv.EnvBuilder().create(self.path)
-            else:
-                venv.EnvBuilder(with_pip=True).create(self.path)
+        venv.EnvBuilder(with_pip=True).create(self.path)
 
         env_scripts = self._get_env_path('scripts')
         if not env_scripts:
             raise RuntimeError("Couldn't get environment scripts path")
-
-        exe = 'python'
+        exe = 'pypy3' if platform.python_implementation() == 'PyPy' else 'python'
```

Comment:
`python` is optional, and while virtualenv creates it, venv only does the minimal and creates only `pypy3.exe`.